build.gradle

```diff
@@ -36,7 +36,7 @@ buildscript {
         }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.0.2'
+        classpath 'com.android.tools.build:gradle:7.0.4'
```

Comment:
Can you revert this? We should track changes in individual PRs/commits in case something breaks due to this.  (throwback to #2833)